buildout.cfg

```diff
@@ -15,7 +15,7 @@ parts +=
     rebuild_i18n-sh
 
 [code-analysis]
-recipe = plone.recipe.codeanalysis[recommended]
```

Comment:
@rodfersou don't do this, pin the plugin that is failing to the version that was working before.